### PR TITLE
Video: ZAP Chat 15 + latest / featured vids

### PR DESCRIPTION
**Pull request by psiinon**
Added new ZAP Chat video.
Also added it as the "Featured" video on the ZAP Chat page and the "Latest" one on the Videos page..

### CODE REVIEW

File: site/content/videos.md [hunk 1]
```diff
@@ -33,3 +33,5 @@ groups:
         link: /videos-list/
         desc: All of the Official ZAP videos, tagged and searchable
 ---
+### Latest Video
+{{<youtube uuid="s8DKKTTHsGM" small="true">}}
```
Comment (kingthorin) [inline site/content/videos.md:37]: Hmmm can we out a numeric id on them or pick by date instead of having to set it manually?

(Not a blocker)

**Review by kingthorin — APPROVED** [review]

File: site/content/videos.md (hunk 1 above)
Comment (psiinon) [reply site/content/videos.md:37]: Good thinking - we could pick the first one in https://github.com/zaproxy/zaproxy-website/blob/main/site/data/videos.yaml - let me try that...

Comment (psiinon) [reply site/content/videos.md]: Now renders the latest youtube vid 😁

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**